tests/components/rfxtrx/test_light.py

```diff
@@ -83,6 +85,27 @@ async def test_one_light(hass, rfxtrx):
     ]
 
 
+@pytest.mark.parametrize("state,brightness", [["on", 100], ["on", 50], ["off", None]])
+async def test_state_restore(hass, rfxtrx, state, brightness):
+    """State restoration."""
+
+    entity_id = "light.ac_213c7f2_16"
+
+    mock_restore_cache(
+        hass, [State(entity_id, state, attributes={ATTR_BRIGHTNESS: brightness})]
+    )
+
+    assert await async_setup_component(
+        hass,
+        "rfxtrx",
+        {"rfxtrx": {"device": "abcd", "devices": {"0b1100cd0213c7f210020f51": {}}}},
+    )
+    await hass.async_block_till_done()
+
+    assert hass.states.get(entity_id).state == state
+    assert hass.states.get(entity_id).attributes.get(ATTR_BRIGHTNESS) == brightness
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)